brayns/common/types.h

```diff
@@ -335,6 +340,9 @@ enum class CameraMode
     inspect
 };
 
+// A clip plane is defined by a normal and a distance to the center of the scene bounding box
+typedef std::vector< Vector4f > ClipPlanes;
```

Comment:
This definition packs the plane equation into 4 floats for implementation convenience. That makes it easier to transfer the data to the underlying rendering engine. What matters is what is exposed to the client: the definition in the fbs, and this is common to RTNeuron, Livre and Brayns.